src/catalog/src/builtin.rs

```diff
@@ -2132,6 +2133,19 @@ pub static MZ_COMPUTE_OPERATOR_HYDRATION_STATUSES_PER_WORKER: LazyLock<BuiltinSo
         access: vec![PUBLIC_SELECT],
     });
 
+pub static MZ_CATALOG_RAW: LazyLock<BuiltinSource> = LazyLock::new(|| BuiltinSource {
+    name: "mz_catalog_raw",
```

Comment:
`_raw` was meant as a bit of a placeholder when I typed it. Happy to go with whatever, it's easy to change later, but just in case we like any of these better, some other options I've considered are `_json`, `_unified`, `_all`, `_multiplexed`, `_multi`, `mz_internal.mz_catalog`

Reply:
I also thought about other names, and decided I liked `raw` best. And indeed it's very easy to change given that this is `mz_system` only.